test/terraform_aws_simple_test.go

```diff
@@ -0,0 +1,67 @@
+// =================================================================
+//
+// Work of the U.S. Department of Defense, Defense Digital Service.
+// Released as open source under the MIT License.  See LICENSE file.
+//
+// =================================================================
+
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"github.com/aws/aws-sdk-go/service/lambda"
```

Comment:
Please add a space between the golang native imports and the imported imports from github.com.